fe/fe-core/src/main/java/com/starrocks/http/HttpRequestException.java

```diff
@@ -0,0 +1,28 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
need replace all license header by this when you create a new file
// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.